dazzle.yaml

```diff
@@ -167,6 +167,11 @@ combiner:
         - tool-nginx
         - tool-nix:2.11.0
         - tool-yugabytedb:2.19
+    - name: bun
+      ref:
+      - full
```

Comment:
@axonasif would it make sense if we based this off of `base` instead in order to create a slim image like we have for Node.js? 👀

Reply:
@filiptronicek that's a good idea. If they want to base it off of `full`, I'd argue they should create and maintain a base image of their own, because it sounds like they have a specific use case (for depending on tech other than bun and what's in base).